modules/k8s_cluster/README.md

```diff
@@ -41,4 +41,4 @@ No modules.
 |------|-------------|
 | <a name="output_client_id"></a> [client\_id](#output\_client\_id) | The client\_id that was used to onboard the k8s cluster, and may be used to onboard other clusters. |
 | <a name="output_client_secret"></a> [client\_secret](#output\_client\_secret) | The client\_secret that was used to onboard the k8s cluster, and may be used to onboard other clusters. |
-<!-- END_TF_DOCS -->
+<!-- END_TF_DOCS -->
```

Comment:
Weird that github is showing a difference here? Is that due to the change to replace from inject in the GHA?

Reply:
Probably, yes.